### DISCUSSION TITLE
1.1 bugs

### DISCUSSION THREAD

**Discussion by andivax** [apohl79/audiogridder#353] [Help]
HI!
1.1 looking much more stable than 1.1 betas, Thank you.
Still found some bugs. Reaper 6.20 x64 Win 10 20H2.

Tested plugin and server on ONE machine.

1) Nebula 4 doesn't work. After loading N4 Audiogridder's insert slot blinks.
2) Can't find SAVE PRESET function: https://prnt.sc/yb6plv
3) Still 2 servers instead one in the servers list: https://prnt.sc/yb6ol4
4) can't USE Audiogridder plugin window, while plugin inserted in Audiogridder and plugins GUI is opened.
5) is it possible to add CLOSE button (X) to plugins hosted by Audiogridder? Sometimes opened plugins GUI overcast Audiogridder's GUI: https://prnt.sc/yb6m9c
6) move up/down and delete, bypass, edit, hide menu functions is absolete. no need to double it IMHO.
7) can you make disabled plugins less contrast? or highlight active plugins with some additional background? ATM it's not very easy to understand which plugin is active and which is not for people with bad vision or problematic sight.
8) please add posibility to save ALL loaded plugins in ONE Audiogridder instance as CHAIN PRESET or something like that. This will be very usefull for re-calling complex setups.
9) automation of multipple (2 or more) Acustica Audio Acqua plugins parameters causes clicks and crackles. If I am tweaking these parameters from plugin's GUI there is NO clicks/crackles.

**Comment by apohl79** [comment]
Thx for your suggestions!

> 1. Nebula 4 doesn't work. After loading N4 Audiogridder's insert slot blinks.

You can submit a separate bug report including logs/traces etc. But it is what it is with acustica plugins, some work others cause issues. Their aqua stuff mostly works I think but N4 was always a problem. As long as acustica stays unsupportive, I can't do much about it, as it is non-free software.

> 2. Can't find SAVE PRESET function: https://prnt.sc/yb6plv

This menu does only reflect presets a plugin is reporting through the plugin interface. It's no custom preset menu. I will add AG preset support later in the upcoming v1.2.

> 3. Still 2 servers instead one in the servers list: https://prnt.sc/yb6ol4

It is either because of multiple network interfaces or ipv4 and ipv6 addresses I guess. A way to distinguish between different entries with the same name is needed. I will add a feature request ticket for this.

> 4. can't USE Audiogridder plugin window, while plugin inserted in Audiogridder and plugins GUI is opened.
> 5. is it possible to add CLOSE button (X) to plugins hosted by Audiogridder? Sometimes opened plugins GUI overcast Audiogridder's GUI: https://prnt.sc/yb6m9c

I will add a feature request ticket for this.

> 6. move up/down and delete, bypass, edit, hide menu functions is absolete. no need to double it IMHO.

makes sense as well.

> 7. can you make disabled plugins less contrast? or highlight active plugins with some additional background? ATM it's not very easy to understand which plugin is active and which is not for people with bad vision or problematic sight.

got it.

> 8. please add posibility to save ALL loaded plugins in ONE Audiogridder instance as CHAIN PRESET or something like that. This will be very usefull for re-calling complex setups.

This will be possible with AG presets as mentioned in (2).

> 9. automation of multipple (2 or more) Acustica Audio Acqua plugins parameters causes clicks and crackles. If I am tweaking these parameters from plugin's GUI there is NO clicks/crackles.

Can you try the generic editor? Does it cause crackles too?

**Reply by andivax** [reply]
thank you Andreas!
I will speak with Giancarlo from Acustica Audio.
Actually AA plugins is the reason for us to use your amazing product! ))

**Comment by andivax** [comment]
Andreas, I have created a new topic about Acustica Audio N4 with LOGs and TRACEs.
Thanx!

**Reply by apohl79** [reply]
Thx!

**Comment by andivax** [comment]
AA is unable to help you at this moment.
They are busy with M1 porting
Unfortunately (

Пт, 5 февр. 2021 г. в 22:36, Andreas Pohl <notifications@github.com>:

> Thx!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/353#discussioncomment-342913>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARLSQUKIFBC2CLJM4RB43PDS5RJEBANCNFSM4XDPVYSQ>
> .
>
-- 
www.ANDIVAXMASTERING.com

**Comment by apohl79** [comment]
Ok, thx for trying
